### PR TITLE
CU-yde7jq_Replace-Signature-Processor-with-the-new-SigParser-from-mutt-data_Ben-Brown

### PR DESCRIPTION
**Pull request by thehandcoder**
Using the new sig parser instead of the old one.

### CODE REVIEW

**Comment by thehandcoder** [comment]
Task linked: [CU-yde7jq Replace Signature Processor with the new SigParser from mutt data](https://app.clickup.com/t/yde7jq)

File: exports/export_wc_orders.php [hunk 1]
```diff
@@ -378,6 +376,8 @@ function export_wc_create_order($order, $reason)
                 'last_name' => $last_name,
             ]
         );
+
+        return;
```
Comment (Hobstweedl) [inline exports/export_wc_orders.php:380]: why is this return here?

**Review by Hobstweedl — APPROVED** [review]

File: exports/export_wc_orders.php (hunk 1 above)
Comment (thehandcoder) [reply exports/export_wc_orders.php:380]: Because if we don't have a patient_id_wc, calling the url that requires a patient_id_wc will result in an error?

Comment (thehandcoder) [reply exports/export_wc_orders.php:380]: This is a hotfix from main that got mixed in with my merge to develop

**Pull request merged**